src/components/approval-type/ApprovalType.vue

```diff
@@ -1,34 +1,70 @@
 <template>
   <div id="approval-type">
     <v-row no-gutters>
-      <v-col cols="12" sm="3" class="pr-4">
+      <v-col cols="12" sm="3" class="pr-4 pt-5">
```

Comment:
This top padding (or margin) should be in the parent. That way, different parents can add different amounts of whitespace.

In a nutshell: padding and margins should be "tight" around a component.

Reply:
Can I only change the padding for this label in the parent? This top padding is used to make the "Approval Type" label on the same line as the first radio button. If I don't have it here, how can I only change it's padding in the parent?

Reply:
This is what it would look like:
![without pt](https://user-images.githubusercontent.com/122301442/219714810-bdf0ae0c-dd83-44d0-9498-14877691d744.PNG)

Reply:
Actually, I have a better idea. I'll just remove this entire label and have it in the parent. So the shared component doesn't include this label. What do you think?

Reply:
It would best for this component to be consistent with the other components. I think that means the label should be in the component.

So what's adding the extra whitespace above the radio buttons? Inspect the elements and tweak the offending one. Let me know if I can help.

Reply:
![fixed padding issue](https://user-images.githubusercontent.com/122301442/219792619-eb1e5028-23ee-4082-9fa7-7b11cd37fe1f.PNG)
Fixed! No more extra whitespace above the radio buttons.